router-bridge/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "router-bridge"
-version = "0.6.0-beta.0+v2.9.0-beta.0"
+version = "0.6.0-beta.0+v2.9.0-beta.1"
```

Comment:
(partial review, still looking at code) Note that this should be `0.6.0-beta.1+v2.9.0-beta.0`, not `0.6.0-beta.0+v2.9.0-beta.1`.

Reply:
Ok, I can update that. Per @SimonSapin's comment, should I rebase this and target `main`? Or is it ok to release a `beta.1` from this branch?

Reply:
I think we've been releasing RCs off branches and not `main` (or at least, recently we have been).

Reply:
I realized I'm not super familiar with how we release this package. Is the branch name significant, or is it fine to merge here after approval and publish from this branch?